openProvider.md

```diff
@@ -0,0 +1,34 @@
+我们可以根据Pulsar
```

Comment:
Is this file needed?

Reply:
Oh, this is a draft I wrote when I responded to Pulsar's external interface, and I accidentally committed it. Thanks for pointing out this issue